### PR TITLE
Developer  role

### PR DESCRIPTION
**Pull request by dsavchenko**

### CODE REVIEW

**Review by volodymyrss — APPROVED** [review]
looks good to me! But I did not look very deep.
I suggest to ask @burnout87 for a review since he's been doing these kg integrations and he's might be available now.

File: tests/test_plugin.py [hunk 1]
```diff
@@ -231,3 +251,52 @@ def test_image_product(dispatcher_live_fixture, mock_backend):
     assert c.status_code == 200
     imdata = jdata['products']['numpy_data_product_list'][0]
     oda_ndp = ImageDataProduct.decode(imdata)
+
+def test_default_kg(conf_file, dispatcher_live_fixture):
```
Comment (burnout87) [inline tests/test_plugin.py:255]: is `conf_file` needed for this test?

File: tests/test_plugin.py [hunk 2]
```diff
@@ -231,3 +251,52 @@ def test_image_product(dispatcher_live_fixture, mock_backend):
...
+                              'token': encoded_token})
+    logger.info("content: %s", c.text)
+    jdata = c.json()
+    logger.info(json.dumps(jdata, indent=4, sort_keys=True))
+    logger.info(jdata)
+    assert c.status_code == 200 
+    assert 'lightcurve-example' in jdata # TODO: change to what will be used in docs
+
+@pytest.mark.parametrize("privileged", [True, False])
+def test_local_kg(conf_file, dispatcher_live_fixture, privileged):
```
Comment (burnout87) [inline tests/test_plugin.py:270]: where do you define `conf_file` ?

**Review by burnout87 — COMMENTED** [review]
The querying part looks good to me. However some parts are hard-coded (as you pointed), but will be for sure changed in the future.

File: tests/test_plugin.py (hunk 1 above)
Comment (dsavchenko) [reply tests/test_plugin.py:255]: Of course not, thanks

File: tests/test_plugin.py (hunk 2 above)
Comment (dsavchenko) [reply tests/test_plugin.py:270]: It's here https://github.com/oda-hub/dispatcher-plugin-nb2workflow/blob/0f66d8ccd6e3a106186d10bc7524f88a109efb43/tests/conftest.py#L44

**Comment by dsavchenko** [comment]
> However some parts are hard-coded (as you pointed), but will be for sure changed in the future.

Yes, I just kept the default as before, it was already hard-coded.

**Review by burnout87 — APPROVED** [review]

**Pull request merged**